charts/stable/channels-dvr/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+appVersion: "latest"
+dependencies:
+  - name: common
+    repository: https://library-charts.truecharts.org
+    version: 11.0.3
+deprecated: false
+description: Channels DVR Server
+home: https://truecharts.org/docs/charts/stable/channels-dvr
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/channels-dvr.png
+keywords:
+  - channels-dvr
+kubeVersion: ">=1.16.0-0"
+maintainers:
+  - email: info@truecharts.org
+    name: TrueCharts
+    url: https://truecharts.org
+name: channels-dvr
+sources:
+  - https://github.com/truecharts/charts/tree/master/charts/stable/channels-dvr
+  - https://hub.docker.com/r/fancybits/channels-dvr
+type: application
+version: 1.0.0
+annotations:
+  truecharts.org/catagories: |
+    - media
+  truecharts.org/SCALE-support: "true"
+  truecharts.org/grade: U
```

Comment:
```yaml
apiVersion: v2
appVersion: "latest"
dependencies:
  - name: common
    repository: https://library-charts.truecharts.org
    version: 11.0.3
deprecated: false
description: Channels DVR Server
home: https://truecharts.org/docs/charts/incubator/channels-dvr
icon: https://truecharts.org/img/hotlink-ok/chart-icons/channels-dvr.png
keywords:
  - channels-dvr
kubeVersion: ">=1.16.0-0"
maintainers:
  - email: info@truecharts.org
    name: TrueCharts
    url: https://truecharts.org
name: channels-dvr
sources:
  - https://github.com/truecharts/charts/tree/master/charts/incubator/channels-dvr
  - https://hub.docker.com/r/fancybits/channels-dvr
type: application
version: 0.0.1
annotations:
  truecharts.org/catagories: |
    - media
  truecharts.org/SCALE-support: "true"
  truecharts.org/grade: U
```

Reply:
links doesn't really matter, They get updated automatically ;)

Reply:
i was changing the version mostly, thats good to know that it was implemented!

Reply:
@Xstar97 Done in https://github.com/truecharts/charts/pull/4527/commits/d16faf0e594a9d1b96079f2f82f547fce86c3aa5